server/etcdserver/api/membership/cluster.go

```diff
@@ -783,11 +783,7 @@ func (c *RaftCluster) IsLocalMemberLearner() bool {
 	defer c.Unlock()
 	localMember, ok := c.members[c.localID]
 	if !ok {
-		c.lg.Panic(
-			"failed to find local ID in cluster members",
-			zap.String("cluster-id", c.cid.String()),
-			zap.String("local-member-id", c.localID.String()),
-		)
```

Comment:
Suggest not to change this. I don't think it will happen in production or test environment. If it happens, it means something critical occurs.